src/test/groovy/synacy/gradprogram/spock/exercise/DateUtilsSpec.groovy

```diff
@@ -0,0 +1,34 @@
+package synacy.gradprogram.spock.exercise
+
+import com.synacy.gradprogram.spock.exercise.DateUtils
+import spock.lang.Specification
+
+import java.time.LocalDate
+import java.time.ZoneId
+
+class DateUtilsSpec extends Specification {
+
+    DateUtils dateUtils
+
+    void setup() {
+        dateUtils = new DateUtils()
+    }
+
+    def "calculateDifferenceInDays should return 0 for the same dates"() {
```

Comment:
This is okay but for me personally, I think this is testing the same thing as the next unit test :+1: